README.md

```diff
@@ -65,20 +65,21 @@ This repository follows the mono-repository structure. It contains multiple npm
 
 To create a new package, call the `ckeditor5-package-generator` executable file. It requires a single argument which is the package name. It must follow the schema: `@scope/ckeditor5-package`, where [@scope](https://docs.npmjs.com/about-scopes) is an owner of the package, and `ckeditor5-package` is the package name. It must start with the `ckeditor5-` prefix.
 
-The tool will create a new directory called `ckeditor5-package` with an example plugin called `MyPlugin` and tools for developing the package.
+The tool will create a new directory called `ckeditor5-package` with an example plugin using the provided package name and tools for its development.
```

Comment:
Does the name include `ckeditor5` prefix? If it doesn't, we should mention that, in fact, the plugin will be named `Package`.

Reply:
The directory name will include the `ckeditor5` prefix, but the plugin name itself will not, just as it was before this PR.